.github/workflows/test-ci.yml

```diff
@@ -202,21 +162,18 @@ jobs:
           path: ~/.cache/brainweb
           key: ${{ runner.os }}-Brainweb
 
-      - name: Install Python deps
-        shell: bash
-        run: |
-          python -m pip install --upgrade pip
-          python -m pip install -e .[test,dev]
-          python -m pip install finufft pooch brainweb-dl torch fastmri
-        
-      - name: Install GPU related interfaces
+      - name: Point to CUDA 12.1 #TODO: This can be combined from other jobs
```

Comment:
I don't get your comment, what combinaison do you have in mind ?

Reply:
We do this even in other places in the CI, we not really re-do it again and again